helpers/notificationHelper.ts

```diff
@@ -43,6 +44,26 @@ export async function createNewReviewOnYourLevelNotification(levelUserId: string
   });
 }
 
+export async function completedXAmountOfLevelsNotification(userIdWhoBeatXAmountOfLevels: string | ObjectId, message?: string | Object){
```

Comment:
I think it would be better to generalize the notification as a AchievementNotification...

Reply:
Basically whenever a new achievement is added to a user we should create the notification